src/main/java/uk/gov/digital/ho/proving/income/audit/AuditClient.java

```diff
@@ -90,13 +90,15 @@ private void dispatchAuditableData(AuditableData auditableData) {
             .toUri();
 
         HttpEntity<Void> entity = new HttpEntity<>(generateRestHeaders());
-        ResponseEntity<List<AuditRecord>> auditRecords = restTemplate.exchange(uri, GET, entity, new ParameterizedTypeReference<List<AuditRecord>>() {});
+        ResponseEntity<List<AuditRecord>> auditRecords = restTemplate.exchange(uri, GET, entity, new ParameterizedTypeReference<List<AuditRecord>>() {
+        });
```

Comment:
You've not changed this line but it's been altered, probably by IntelliJ's auto-formatter. Could you please put it back to how it was as I prefer this on a single line? There's an IntelliJ setting called something like "allow simple classes on a single line" which you might want to toggle to allow that.

Please go through and find any other lines that you haven't altered but are rearranged too.

Reply:
Resolved.